packages/amazon/src/serverGroup/configure/wizard/instanceType/advancedMode/advancedMode.less

```diff
@@ -0,0 +1,110 @@
+.advanced-mode-selector {
+  .instance-profile-header {
+    display: inline-block;
+    button {
+      width: 100%;
+    }
+    h4 {
+      min-height: 52px;
+    }
+    .panel-heading {
+      padding: 0 0 0 0;
+    }
+    &.profile-button {
+      width: calc(~'25% - 20px');
+      margin: 0 10px 15px 10px;
+    }
+  }
+
+  .sub-section {
+    padding: 0 15px;
+    margin-bottom: 5px;
+
+    .description {
+      padding: 0 10px;
+    }
+
+    tr {
+      &.unavailable {
+        opacity: 0.5;
+      }
+
+      &.sortable {
+        .instance-type-drag-handle {
+          display: inline-block;
+          opacity: 0.9;
+          font-size: 100%;
+          padding-top: 8px;
+        }
+      }
+    }
+
+    th {
+      font-size: 110%;
+    }
+  }
+}
+
+.custom-profile {
+  .select {
+    width: 80%;
+  }
+}
+
+.instance-profile {
+  vertical-align: top;
+  text-align: left;
+  min-height: 100px;
+  font-size: 90%;
+  border: 1px solid var(--color-silver);
+  border-radius: 3px;
+  color: var(--color-mineshaft);
+  background-color: transparent;
+
+  &:hover,
+  &:focus {
+    color: var(--color-mineshaft);
+    text-decoration: none;
+  }
+
+  .selected-indicator {
+    display: block;
```

Comment:
Same comment about `flex`

Reply:
Changing styling will change the look-and-feel of `advanced mode` compared to `simple mode`(previous version). Discussed offline and agreed not to use flex.